corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -480,32 +487,51 @@ hqDefine("geospatial/js/geospatial_map", [
             }
         }
 
-        function loadCases(caseData) {
+        // This function partitions case data into cases with and without GPS data
+        function partitionCaseData(caseData) {
+            const caseItem = {
+                ID: 0,
+                COORDINATES: 1,
+                LINK: 2,
+            };
```

Comment:
nit: making an object for caseItem would be better than storing indexes separately in constants.